src/models/User.d.ts

```diff
@@ -1,22 +1,25 @@
 import { Document, Model } from "mongoose";
 
 interface IUser {
-  id: string
-  userName: string
-  token: string
+  id: string;
+  userName: string;
+  token: string;
 }
 
 interface IProfile {
-  id: string
-  displayName: string
-  [key: string]: any
+  id: string;
+  displayName: string;
+  [key: string]: any;
 }
 
-interface IUserDoc extends Document, IUser {
-}
+interface IUserDoc extends Document, IUser {}
 
 interface IUserModel extends Model<IUserDoc> {
-  findOneOrCreate(accessToken: string, profile: IProfile): Promise<IUserDoc>
+  findOrCreate(
+    arg0: { googleId: string },
+    arg1: (err: any, user: any) => any
```

Comment:
`any` type is not acceptable.
`arg0`, `arg1` are you sure this is the best variable name?

Reply:
Do you have any particular sugestion for naming?